api/src/main/resources/workbench.yaml

```diff
@@ -91,6 +91,32 @@ paths:
           schema:
             $ref: "#/definitions/UsernameTakenResponse"
 
+  /api/v1/create-account:
```

Comment:
Some inconsistency in our REST paths... above we have v1/sendBugReport (camelCase) and here you have create-account, delete-account, is-username-taken.

I think REST paths normally don't have hyphens... can we settle on camelCase?

Reply:
Actually, I'd like us to move to hyphen-separated lowercase for all URLs (I'll follow up with a separate PR). URLs and things derived from them tend to show up in a browser's address bar. That is user-facing. A non-developer does not appreciate the difference between underscore vs. space (and the underscore is hard to see) and case-sensitivity.

Reply:
None of our API URLs should show up in an address bar, and non-developers shouldn't be entering any of these URLs into anything; mainly we should worry about what developers will think of our URLs.

I thought camelCase was more standard than using hyphens or underscores, but random Google searching suggests that to the extent there is a standard, hyphens are the way to go... so fine by me. +1 to fixing in a follow-up.

Reply:
Sweet. You're right about the URLs technically, but in my experience URL-like things tend to leak into UI components, so to be safe I like to treat them with similar usability principles even in APIs.